module/observations/helper.js

```diff
@@ -217,7 +217,8 @@ module.exports = class ObservationsHelper {
     static createObservation(data,userId,solution,userRoleInformation="",userProfileInformation = {}) {
         return new Promise(async (resolve, reject) => {
             try {
-    
+                
+                let overWriteUserInfoIfReferenceFromProjectsIsFound = false;
```

Comment:
@borkarsaish65 Don't use new flag just do it for if data.project is there

Reply:
fixed